javagen/src/main/java/com/azure/autorest/Javagen.java

```diff
@@ -31,8 +31,12 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 
+import static com.azure.autorest.model.clientmodel.ClientMethodType.PagingSync;
+import static com.azure.autorest.model.clientmodel.ClientMethodType.SimpleSyncRestResponse;
```

Comment:
Suggestion is to avoid static import, if not really necessary.

Reply:
Done